### DISCUSSION TITLE
Moving of SolidOS to new GitHub organisation

### DISCUSSION THREAD

**Discussion by timea-solid** [solid/solidos#150] [Archive] [answered]
This is a list and discussion about moving the SolidOS stack to the new organization: https://github.com/SolidOS
Todo and to consider:

- [x] invite current active contributors to the new organization
- [x] create an admin group that contains owners -> do we want to have more owners?

Links we need to be careful to change:

- [x] SolidOS package.json  needs to update all github.com/solid/.... links
- [x] SolidOS `add` script needs to update all github.com/solid/.... links
- [x] SolidOS README needs to update all github.com/solid/.... links
- [x] SolidOS discussions -> can they be moved?
- [x] SolidOS documentation -> change multiple occurrences of github.com/solid/solidos and github.com/solid
- [x] can wiki be moved? YES ([see this](https://docs.github.com/en/enterprise-server@3.3/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository))
- [x] can project board be moved? 
- [x] How about the link of the issues? (YES ([see this](https://docs.github.com/en/enterprise-server@3.3/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository)))
- [x] GitHub pages referenced outside need to change. (the Github Pages themselves will be automatically changed)
- [x] Solidos.solidcommunity.net -> change links referencing  github.com/solid/solidos
- [x] solidproject.org -> any pointers to SolidOS? (need to be changed)
- [ ] npm to be considered later if there is any move happening
- [x] mashlib build-dev.sh needs to update all github.com/solid/.... links
- [x] mashlib local-webpack needs to update all github.com/solid/.... links
- [x] mashlib package.json  needs to update all github.com/solid/.... links
- [x] mashlib README has 2 links to github.com/solid/solidos and multiple github.com/solid/.... links
- [x] pane-registry README needs to update all github.com/solid/.... links
- [x] pane-registry package.json  needs to update all github.com/solid/.... links
- [x] solid-logic package.json  needs to update all github.com/solid/.... links
- [x] solid-panes header.js ->  change link of drop down menu regarding where to report issues and link to user guide
- [x] solid-panes package.json  needs to update all github.com/solid/.... links
- [x] solid-panes README needs to update all github.com/solid/.... links
- [x] solid-panes basicPreferences -> needs to update all github.com/solid/.... links
- [x] solid-ui needs to update all github.com/solid/.... links in README, package.json, some documentation files

Is anything missing? Any further considerations needed please add.

**Comment by joachimvh** [comment]
If you want to move the gitter, you can contact them to have all the messages moved to the new channel once you make one: https://gitlab.com/gitterHQ/webapp/-/blob/develop/docs/faq.md#transfer-repo-to-a-new-org

We did this for the CSS move and it took about one week for them to do it. Not sure if you can already use the new channel during that time, I decided to only make the new channel public after the move. After the move the old channel will be completely empty as you can see in our old channel: https://gitter.im/solid/community-server

**Comment by jeff-zucker** [comment]
> Is anything missing? 

Data-Kitchen?

**Comment by timea-solid** [comment]
- [x] User guide also needs to move ->github pages link changes
- [x] user guide README links need update

**Comment by timea-solid** [comment]
- [x] forms-playground update README links
- [x] https://github.com/SolidOS/form-playground/blob/main/playground.html#L8 needs changing of mashlib usage link

**Comment by timea-solid** [comment, marked as answer]
- [x] rdflib links

**Comment by timea-solid** [comment]
- [ ] user guide has link to NSS (in case it changes needs update)

**Comment by timea-solid** [comment]
- [ ] gitter chat channel links need update in SolidOS Readme when we change Gitter
